cala-ledger/src/account_set/cursor.rs

```diff
@@ -20,3 +20,9 @@ impl From<&AccountSetValues> for AccountSetByNameCursor {
 pub struct AccountSetMemberCursor {
     pub member_created_at: chrono::DateTime<chrono::Utc>,
 }
+
+impl From<chrono::DateTime<chrono::Utc>> for AccountSetMemberCursor {
```

Comment:
This should be from a domain type to another not from a primitive. so `From<AccountSetMember>`